uxarray/grid.py

```diff
@@ -225,3 +227,34 @@ def __init_ds_var_names__(self):
             "nMesh2_face": "nMesh2_face",
             "nMaxMesh2_face_nodes": "nMaxMesh2_face_nodes"
         }
+
+    def __init_ds_var_methods__(self):
```

Comment:
How about `__init_grid_var_attrs__` (or simply `__init_var_attrs__` as we are already in `grid.__init__ method`) for the name of this function? I propose this because we are initializing the grid var attributes here, not the dataset var methods.